src/NLog/Common/InternalLogger.cs

```diff
@@ -357,14 +357,14 @@ public static void LogAssemblyVersion(Assembly assembly)
         {
             try
             {
-#if SILVERLIGHT || __IOS__ || __ANDROID__
-                Info(assembly.FullName);
-#else
+#if !SILVERLIGHT && !__IOS__ && !__ANDROID__
```

Comment:
Why this change? (I see nothing got changed functional). It's this a better style?

It will lead to potential conflicts on the coreCLR branch.